blog-posts/boosting-developer-productivity-with-github-actions.md

```diff
@@ -0,0 +1,282 @@
+
+![A release created by github-actions](./images/github-actions/release.jpg)
+
+### Workflow: Stale issues
+As part of the issue management we need to keep track of old issues.
+
+#### What problem does it solve?
+As our repository grows and evolves so does the number of issues that accumulate. Over time some of these issues become obsolete.
+
+#### How does it solve the problem?
+Using a scheduled GitHub Workflow that runs once every night we mark "old" issues (90 days without interaction) as stale. This workflow uses the official `actions/stale` action. If a stale issue is not being interacted with within 5 days it will get closed automatically. If an issue gets marked as stale and the team agrees that it's still relevant during the next refinement meeting, we remove the `stale` label.
+
+![github-actions marking an issue as stale](./images/github-actions/stale-issue.jpg)
+
+### Workflow: Release comments
+We use keywords in Pull Requests like `closes`, `fixes`, or `resolves` to automatically close issues that are done from a development point of view.
+
+#### What problem does it solve?
+With issues being closed when a corresponding Pull Request got closed it's sometimes hard to keep track of the release that a certain fix or feature got published with.
+
+#### How does it solve the problem?
+This GitHub Workflow leaves a comment on released issues that informs any subscriber about the specific release that shipped the changes necessary to close the issue. This is done using a custom bash script that reads the CHANGELOG and extracts all issues that got published as part of the latest release. This list of released issues is then being passed to a custom action that uses GitHub GraphQL API to leave comments on these issues.
+
+![A comment by github-actions that points to the corresponding release](./images/github-actions/release-comment.jpg)
+
+### Workflow: Housekeeping
+Housekeeping is an important aspect of the fast-paced frontend world. We used to have an issue to remind us to update our dependencies every once in a while. As with a lot of manual processes we forgot to do this from time to time. Especially when it comes to important vulnerability fixes it’s important to keep up to date. We now have a workflow to take care of that once a week using a scheduled event. With integrated tools like "Dependabot" this workflow might be obsolete soon, but so far we enjoy the fact that we are in full control of the workflow and can tweak and configure it to our hearts content.
+
+## Conclusion
+
+In this blog post you learned the most important things about the terminology that you need to get started as well as got a first technical glimpse at what it means to author workflows and actions. If you've made it this far, I really recommend you give it a go. I can't stop myself from smiling whenever I see something that was done by `github-actions [bot]` and I really hope I was able to get you interested in GitHub Actions and automation in general.
```

Comment:
This is the perfect conclusion 🤩 My initial impulse: I would put that text at the very end.
But then I see that this will cause issues with the what's next section. So I'm actual good here and I like the way this blog post is ending now 💫